python/requirements.txt

```diff
@@ -43,6 +43,7 @@ uvicorn==0.16.0
 dataclasses; python_version < '3.7'
 starlette==0.18.0
 aiorwlock
+gradio
```

Comment:
Should Serve always come packaged with gradio? This seems a bit heavyweight for the vast majority of cases. @simon-mo @edoakes what do you think?

Reply:
`setup.py` is the ground truth for `pip install ray[serve]`. `requirements.txt` is only used to setup test CI dependencies